backend/index.js

```diff
@@ -1,138 +1,201 @@
-const express = require('express')
-const app = express()
-const server = require('http').Server(app)
-const io = require('socket.io')(server)
-const cors = require('cors')
-const vorpal = require('vorpal')()
-const bodyParser = require('body-parser')
+const express = require('express');
+const app = express();
+const server = require('http').Server(app);
+const io = require('socket.io')(server);
+const cors = require('cors');
+const vorpal = require('vorpal')();
+const bodyParser = require('body-parser');
 
-const { AppController } = require('./controllers/app.controller')
-const { PizzaDeLOrmeau } = require('./pizzas-providers/parse-pizza-de-l-ormeau')
+const { AppController } = require('./controllers/app.controller');
+const { PizzaProvider } = require('./pizzas-providers/parse-pizza-provider');
 
-const pizzaProvider = new PizzaDeLOrmeau()
-const appCtrl = new AppController(pizzaProvider)
+const pizzaProvider = new PizzaProvider();
+let pizzaParserClass = pizzaProvider.getDefaultProvider();
+let pizzaParser = new pizzaParserClass();
+let appCtrl = new AppController(pizzaParser);
 
-const currentDate = new Date()
+const currentDate = new Date();
 
 // ----------------------------------------------------------------------------
 
-let isServerStarted = false
+let isServerStarted = false;
+let commandsAlreadyRegistered = false;
 
-appCtrl
-  .parsePizzaProvider()
-  .then(_ => {
-    app.use(bodyParser.json())
+// We instanciate the whole server
```

Comment:
lowercase comments please

Reply:
Done for whole index.js + default-parser.js
When there is no end point the comment is put in lower case.